controllers/db_participation_operations.py

```diff
@@ -0,0 +1,57 @@
+import sys
+from app import client, db
+from .json_controller import *
+from .db_event_operations import MongoManageEvents
+
+event_manager = MongoManageEvents()
+
+
+# TODO: error handling
+class MongoManageParticipations:
+
+    def __init__(self):
+        self.events_col = db.events
+        self.participation_col = db.participations
+
+    def get_participators(self, event_id):
+        # get participators of given event
+        if event_id is not None:
+            participations = self.participation_col.find_one({"event_id": event_id})
+            return to_json(format_ObjectId(participations))
+        # get all participators for all events
+        else:
+            participation_cursor = self.participation_col.find()
+            return cursor_to_json(participation_cursor)
+
+    def join(self, event_id, user_id):
+        self.participation_col.update_one(
+            {'event_id': event_id},
+            {'$addToSet': {'participators': user_id}},
+            upsert=True
+        )
+
+        # set nr of volunteers on event level
+        event_manager.update_event_participators(event_id)
+        return self.get_participators(event_id)
+
+    def leave(self, event_id, user_id):
+        self.participation_col.update_one(
+            {'event_id': event_id},
+            {'$pull': {'participators': user_id}}
+        )
+
+        # set nr of volunteers on event level
+        event_manager.update_event_participators(event_id)
```

Comment:
It is a bit of a coupling here - (like if we change something in the event_manager logic, we would need to remember to update this here too if needed) - I wouldn't know how to do otherwise to avoid cross service dependencies - maybe call them sequentially (or async :D) on the controller level could be an option - like putting in here: 
https://github.com/Volunteero/events-service/blob/7b1c90bd204ddd14dcea88c86f56e96ec74ea0e9/controllers/db_participation_operations.py#L37-L44  
  
But I am not certain myself, _don't touch it if it works_ the wisdom goes, right? :thinking: